pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -130,24 +130,20 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	})
 
 	It("should allow Clock without Timer", func() {
-		vmi := api.NewMinimalVMI("testvmi")
-		vmi.Spec.Domain.Clock = &v1.Clock{
-			ClockOffset: v1.ClockOffset{
-				UTC: &v1.ClockOffsetUTC{
-					OffsetSeconds: pointer.P(5),
+		const offsetSeconds = 5
+		vmi := newBaseVmi(withDomainClock(
+			&v1.Clock{
+				ClockOffset: v1.ClockOffset{
+					UTC: &v1.ClockOffsetUTC{
+						OffsetSeconds: pointer.P(offsetSeconds),
+					},
 				},
 			},
-		}
-		vmiBytes, _ := json.Marshal(&vmi)
+		))
+
+		ar, err := newAdmissionReviewForVMICreation(vmi)
```

Comment:
I believe this change should be in previous commit

Reply:
Sure, I'll squash them.